From 3ab82b46e5cc5175a92c771bc2979582e13fef2b Mon Sep 17 00:00:00 2001
From: Joel Galenson <jgalenson@google.com>
Date: Wed, 14 Aug 2019 09:47:37 -0700
Subject: [PATCH] Do not track the sysroot.

This will help reproducible builds, as the sysroot depends on the
working directory.
---
 src/librustc/session/config.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/librustc/session/config.rs b/src/librustc/session/config.rs
index b5e3c4cda0..a3382c6290 100644
--- a/src/librustc/session/config.rs
+++ b/src/librustc/session/config.rs
@@ -397,7 +397,7 @@ top_level_options!(
         output_types: OutputTypes [TRACKED],
         search_paths: Vec<SearchPath> [UNTRACKED],
         libs: Vec<(String, Option<String>, Option<cstore::NativeLibraryKind>)> [TRACKED],
-        maybe_sysroot: Option<PathBuf> [TRACKED],
+        maybe_sysroot: Option<PathBuf> [UNTRACKED],
 
         target_triple: TargetTriple [TRACKED],
 
-- 
2.23.0.rc1.153.gdeed80330f-goog

